zk/stages/stages.go

```diff
@@ -110,30 +109,41 @@ func SequencerZkStages(
 			},
 		},
 		{
-			ID:          stages2.IntermediateHashes,
-			Description: "Sequencer Intermediate Hashes",
+			ID:          stages2.HashState,
```

Comment:
Can we keep the ordering of the stages intact here, and we previously ran the `HashState` stage when running with SMT so I think we should keep that as it was rather than return nil, and keep the unwind in place as it was previously.